integration-tests/build.gradle

```diff
@@ -0,0 +1,34 @@
+project.ext.sdkVersion = 23
+project.ext.buildTools = '23.0.0'
+
+// Don't cache SNAPSHOT (changing) dependencies.
+configurations.all {
+    resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
+}
+
+allprojects {
+    def currentVersion = file("${rootDir}/../version.txt").text.trim()
+
+    buildscript {
+        repositories {
+            mavenLocal()
+            jcenter()
+        }
+        dependencies {
+            classpath 'com.android.tools.build:gradle:2.1.0'
```

Comment:
All other projects are using `1.5.0`.

If we move to `2.1.0`, we should update all at the same time.